test/integration/formatting_stack/strategies.clj

```diff
@@ -0,0 +1,153 @@
+(ns integration.formatting-stack.strategies
+  (:require
+   [clojure.java.shell :refer [sh]]
+   [clojure.test :refer [deftest is testing]]
+   [formatting-stack.strategies :as sut])
+  (:import
+   (java.io File)))
+
+(def ^File deletable-file (File. "test-resources/deletable.clj"))
+
+(def ^String createable-filename "test-resources/createable.clj")
+
+(def creatable-contents (pr-str '(ns foo)))
+
+(assert (-> deletable-file .exists))
+
+(assert (not (-> createable-filename File. .exists)))
+
```

Comment:
These should be inside of the test. The files don't have to exist/not-exists when _loading_ this ns, they should when i run the tests

Reply:
> These should be inside of the test.

Note that 2 deftests depend on this assertion. Having 2 deftests fail seems less clear, concise and immediate than having `(refresh)` fail as soon as an issue arises.

Reply:
Except that now i can refresh, delete the file and run the test. yielding unclear failures as well.

Reply:
> Except that now i can refresh, delete the file and run the test.

Running tests without a prior `(refresh)` seems just wrong. Our dev.clj file typically suggests to `(refresh)` prior to running any test